src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3459,19 +3459,12 @@ def test_sql_managed_db_short_retention(self, resource_group, resource_group_loc
             'collation': "Serbian_Cyrillic_100_CS_AS",
             'proxy_override': "Proxy",
             'retention_days_inc': 14,
-            'retention_days_dec': 7
+            'retention_days_dec': 7,
+            'rg': 'v-urmila'
         })
 
-        # Create and prepare VNet and subnet for new virtual cluster
-        self.cmd('network route-table create -g {rg} -n {route_table_name}')
-        self.cmd('network route-table route create -g {rg} --route-table-name {route_table_name} -n {route_name_internet} --next-hop-type Internet --address-prefix 0.0.0.0/0')
-        self.cmd('network route-table route create -g {rg} --route-table-name {route_table_name} -n {route_name_vnetlocal} --next-hop-type VnetLocal --address-prefix 10.0.0.0/24')
-        self.cmd('network vnet create -g {rg} -n {vnet_name} --location {loc} --address-prefix 10.0.0.0/16')
-        self.cmd('network vnet subnet create -g {rg} --vnet-name {vnet_name} -n {subnet_name} --address-prefix 10.0.0.0/24 --route-table {route_table_name}')
-        subnet = self.cmd('network vnet subnet show -g {rg} --vnet-name {vnet_name} -n {subnet_name}').get_output_in_json()
-
         self.kwargs.update({
-            'subnet_id': subnet['id']
+            'subnet_id': '/subscriptions/a8c9a924-06c0-4bde-9788-e7b1370969e1/resourceGroups/v-urmila/providers/Microsoft.Network/virtualNetworks/MIVirtualNetwork/subnets/ManagedInsanceSubnet'
```

Comment:
fixed resource is not recommended in test.

Reply:
The main issue with this is that MI tests are not easily setup. They also run very long when fully setup (>4h). I agree that it should be fixed. My take is that I believe Jovana is not making tests non-recordable, since they usually are, and with or without her change, people would need to do some manual setup/preparing in order to record.

Reply:
discussed with Jovana and Perica offline. They have to make network resource fixed given policy set on their subscription.

is it possible to disable the policy before you run the test and enable it again after done?

Reply:
discussed with Jovana. They cannot disable policy.